notebooks/en/rag_with_hugging_face_gemma_elasticsearch.ipynb

```diff
@@ -0,0 +1,6320 @@
+{
```

Comment:
<p>Include a link to google/gemma-2b-it would be nice for users to reference if needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>👍Great idea. Done.</p>